internal/rbd/rbd_util.go

```diff
@@ -532,7 +534,15 @@ func (ri *rbdImage) isInUse() (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	ri.Primary = mirrorInfo.Primary
+
+	if mirrorInfo.State == librbd.MirrorImageEnabled {
+		if !ri.Primary {
```

Comment:
Shouldn't this be `!mirrorInfo.Primary`?

Reply:
:facepalm: yes you are correct. am missed it somehow